atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleDdlTable.java

```diff
@@ -590,4 +607,12 @@ private SqlConnection getCompactionConnection() {
 
         return sqlConnection;
     }
+
+    private static void logDebugOrTrace(@CompileTimeConstant String message, Arg<?>... args) {
+        if (log.isTraceEnabled()) {
+            log.trace(message, Arrays.asList(args), new SafeRuntimeException("provided for stack trace"));
+        } else if (log.isDebugEnabled()) {
+            log.debug(message, Arrays.asList(args));
```

Comment:
What if both are enabled? Is it okay not to log at the debug level? One may be unaware that trace is enabled and try to see the logs at the debug level.